packages/chain-events/src/chains/substrate/storageFetcher.ts

```diff
@@ -211,9 +185,6 @@ export class StorageFetcher extends IStorageFetcher<ApiPromise> {
       ...democracyReferendaEvents,
       ...democracyPreimageEvents,
       ...treasuryProposalEvents,
-      ...bountyEvents,
-      ...technicalCommitteeProposalEvents,
-      ...councilProposalEvents,
```

Comment:
We can also remove signaling proposals (dead code even on the blockchain!)

Reply:
Removed in last commit